### PR TITLE
add chime6 recipe

### PR DESCRIPTION
**Pull request by simpleoier**

### CODE REVIEW

File: egs2/chime6/asr1/run.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+lm_config="conf/train_lm.yaml"
+
+
+use_lm=false
+use_word_lm=false
+word_vocab_size=65000
+
+./asr.sh \
+    --lang en \
+    --bpe_nlsyms "$(< data/nlsyms.txt tr '\n' ',' | sed 's/,$//g')" \
```
Comment (Emrys365) [inline egs2/chime6/asr1/run.sh:26]: Does `data/nlsyms.txt` exist before stage 1? Will this throw an error if the file does not exist?

Comment (simpleoier) [reply egs2/chime6/asr1/run.sh]: Thanks for pointing out this.

**Comment by sw005320** [comment]
LGTM!

Do you have a result?
Can you also upload a model?

**Comment by simpleoier** [comment]
> Do you have a result? Can you also upload a model?

Sure. I'll do it now.

**Comment by simpleoier** [comment]
@sw005320 This PR is ready to merge. Can you check it again?

**Comment by sw005320** [comment]
Thanks, @simpleoier!

**Pull request merged**